dbscripts/xml/upgrade.xml

```diff
@@ -223,7 +223,11 @@
 	</upgrade>
 
 	<upgrade minversion="3.1.0.0" maxversion="3.4.0.5">
-		<migration class="PKP\migration\upgrade\v3_4_0\I9830_FixEmptyUserLocales" />
+		<migration class="PKP\migration\upgrade\v3_4_0\I9830_FixEmptyUserLocales"/>
+	</upgrade>
+
+	<upgrade minversion="3.4.0.0" maxversion="3.4.0.6">
+		<migration class="PKP\migration\upgrade\v3_4_0\I9813_QuickSubmitSubmissionProgressType"/>
```

Comment:
As the current version is `3.4.0-5`, you can move this migration to the tag above.

Reply:
And it does not matter that also 3.1.x.x - 3.3.9.9 would run it too, after the script I7191_SubmissionProgressType has been run :thinking: -- not so clean but...
Or to actually use `<upgrade minversion="3.4.0.0" maxversion="3.4.0.5">`?

Reply:
@jonasraoni, OK now?